### ISSUE TITLE
dcat:Catalog; should foaf:homepage stay mandatoy?

### ISSUE THREAD

**Issue by sabinem** [opendata-swiss/dcat_ap_ch#2]
This is a discussion starter about the following question:
Should the homepage be mandatory for the dcat:Catalog on DCAT-AP-CH? 

In the implementation on opendata.swiss this field is not used. It is only on the dcat:Catalog class, therefore it can't even provide a default for dcat:Datasets. On the DCAT-AP this property is just recommended for catalog. 

So what are opinions on that topic?

**Comment by metaodi** [comment]
I agree with this proposal. Maybe some guideance could be provided for common cases (e.g. what happens if my dataset is derived from some other organisation? How do we choose a publisher for a dataset out of a joint project of two or more organisations?)
